src/sentry/web/frontend/debug/debug_sentry_app_notify_disable.py

```diff
@@ -0,0 +1,40 @@
+from django.http import HttpRequest, HttpResponse
+from django.views.generic import View
+
+from sentry.constants import SentryAppStatus
+from sentry.integrations.notify_disable import get_provider_type, get_url
+from sentry.models import Organization, SentryApp
+
+from .mail import MailPreview
+
+
+class DebugSentryAppNotifyDisableView(View):
```

Comment:
Something is up because I ran this locally and I don't see the "Sentry App Notify Disable" in the debug view dropdown, and it errors when I try to select "Notify Disable"

Reply:
`src/sentry/templates/sentry/debug/mail/preview.html` it needs to be added to this to show in the dropdown, but check out the error